### ISSUE TITLE
Is it possible to use images without bounding boxes as negative examples?

### ISSUE THREAD

**Issue by ralogon** [pierluigiferrari/ssd_keras#183]
I would like to know if it is possible to train models with this implementation with datasets that contain some images without bounding boxes that will be used as negative examples. In affirmative case, how should this information be encoded in the labels.csv.

Thanks in advance for your wonderful work in this repo!

**Comment by pierluigiferrari** [comment]
With the XML and JSON parsers, having images that do not contain any positive ground truth is no problem, but with the CSV parser this doesn't work out-of-the-box, you would have to either modify the parser or just add the images that do not have positive examples to the dataset after the parser stage. That's easy to do, you would just have to make sure to append each image path to the `filenames` list and for each such image append an empty list to the `labels` list.

**Comment by algila** [comment]
Hi,
Is it needed/required training SSD to add in training set some negative example (also called background or unlabeled) to keep the SSD converging more quickly and efficiently ?. 
I see that the model output have an output dimension of Classes + background, so I was asking to my self if the training also need to have background pictures
